example/src/pages/form/formikRadiobutton.jsx

```diff
@@ -112,6 +119,22 @@ export default class FormikRadiobuttonPage extends React.Component{
                 type='checkbox'
               />
             </div>
+
+            <div className='wrapper'>
+              <FormikCheckbox
+                error={false}
+                field={{
+                  name: 'isDisabled',
+                  onChange: () => this.toggleDisabled()
```

Comment:
Aan de Checkbox geef je een value mee en hier niet? bij Field
**value: this.state.disabled,**